x-pack/plugins/serverless_search/public/application/components/web_crawlers/self_managed_web_crawlers_empty_prompt.tsx

```diff
@@ -0,0 +1,228 @@
+                        <EuiFlexItem>
+                          <EuiIcon color="primary" size="l" type="globe" />
+                        </EuiFlexItem>
+                        <EuiFlexItem>
+                          <EuiIcon size="m" type="sortRight" />
+                        </EuiFlexItem>
+                        <EuiFlexItem>
+                          <EuiIcon color="primary" size="l" type={webCrawlersIcon} />
+                        </EuiFlexItem>
+                        <EuiFlexItem>
+                          <EuiIcon size="m" type="sortRight" />
+                        </EuiFlexItem>
+                        <EuiFlexItem>
+                          <EuiIcon color="primary" size="l" type="logoElasticsearch" />
+                        </EuiFlexItem>
+                      </EuiFlexGroup>
+                    </EuiFlexItem>
+                    <EuiFlexItem>
+                      <EuiText>
+                        <p>
+                          {i18n.translate(
+                            'xpack.serverlessSearch.webCrawlersEmpty.guideThreeDescription',
+                            {
+                              defaultMessage:
+                                'Configure your web crawler and connect it to Elasticsearch',
+                            }
+                          )}
+                        </p>
+                      </EuiText>
+                    </EuiFlexItem>
+                  </EuiFlexGroup>
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiPanel>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      }
+      actions={
+        <>
+          <EuiFlexItem>
+            <EuiButton
+              data-test-subj="serverlessSearchEmptyConnectorsPromptCreateSelfManagedConnectorButton"
+              fill
+              iconType={githubIcon}
+              href={'https://github.com/elastic/crawler'}
+              target="_blank"
+            >
+              {i18n.translate('xpack.serverlessSearch.webCrawlersEmpty.selfManagedButton', {
+                defaultMessage: 'Self-managed web crawler',
+              })}
+            </EuiButton>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFlexGroup direction="column" gutterSize="s" alignItems="center">
+              <EuiFlexItem>
+                <EuiButton
+                  data-test-subj="serverlessSearchEmptyConnectorsPromptCreateElasticManagedConnectorButton"
+                  onClick={() =>
+                    navigateToUrl(`${BASE_WEB_CRAWLERS_PATH}/${ELASTIC_MANAGED_WEB_CRAWLERS_PATH}`)
+                  }
+                >
+                  {i18n.translate('xpack.serverlessSearch.webCrawlersEmpty.elasticManagedButton', {
+                    defaultMessage: 'Elastic managed web crawler',
+                  })}
+                </EuiButton>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiBadge color="accent">Coming soon</EuiBadge>
```

Comment:
This should be translated

Reply:
Some comment as above. I moved it to a reusable constant.